GlucoseKit/GlucoseStore.swift

```diff
@@ -124,7 +124,7 @@ public final class GlucoseStore: HealthKitSampleStore {
                     let sortedGlucose = glucose.sorted { $0.startDate < $1.startDate }
 
                     self.sampleDataCache.append(contentsOf: sortedGlucose)
-                    self.purgeOldGlucoseSamples()
+                    //self.purgeOldGlucoseSamples()
```

Comment:
The device field in addGlucose wouldn't be of use would it?  Then if the device is a known to backfill data purge, otherwise don't purge?